app/app.ts

```diff
@@ -77,6 +77,18 @@ app.config(['$stateProvider', '$urlRouterProvider', function ($stateProvider: ng
             controller: 'vendorProfileCtrl',
             authenticate: true
         })
```

Comment:
Maybe a better name for these states would truckProfiles and truckProfiles.details or trucks and trucks.details (just a thought)